storage/tianmu/loader/parsing_strategy.cpp

```diff
@@ -606,9 +606,9 @@ void ParsingStrategy::GetValue(const char *value_ptr, size_t value_size, ushort
 
   } else if (core::ATI::IsTxtType(ati.Type())) {
     // process escape characters
-    if (ati.CharLen() < (uint)value_size) {
+    if (ati.Precision() < (uint)value_size) {
```

Comment:
1: using c++ style type cast. 
2: how about the boundary checking, such as ati.Precision() == value_size.